sources/clarify.toml

```diff
@@ -201,3 +201,23 @@ license-files = [
     { path = "LICENSE-MIT", hash = 0xb9f15462 },
     { path = "LICENSE-APACHE", hash = 0x91d5a0a7 },
 ]
+
+[clarify.aws-lc-sys]
+expression = "ISC AND (Apache-2.0 OR ISC) AND OpenSSL AND MIT"
+license-files = [
+    { path = "LICENSE", hash = 0xf308ccd7 },
+    { path = "aws-lc/LICENSE", hash = 0xb6d14686 },
+    { path = "aws-lc/third_party/fiat/LICENSE", hash = 0x75829ee2 },
+]
+
+[clarify.aws-lc-fips-sys]
+expression = "BSD-3-Clause AND MIT AND GPL-2.0-only"
```

Comment:
I would expect all the license statements for `aws-lc-sys` would also apply to `aws-lc-fips-sys`, with the addition of the `jitterentropy` license.

`jitterentropy` says we can pick either `BSD-3-Clause` or `GPL-2.0-only` so just pick `BSD-3-Clause` and add a comment to that effect.